CMakeLists.txt

```diff
@@ -1,18 +1,18 @@
-cmake_minimum_required(VERSION 3.10)
-
-project ("Xournal++" CXX C)
+cmake_minimum_required(VERSION 3.11)
+cmake_policy(VERSION 3.11)
 
 ## Project related Variables
 ## Also update changelog in debian folder!
-set (CPACK_PACKAGE_VERSION_MAJOR "1")
-set (CPACK_PACKAGE_VERSION_MINOR "1")
-set (CPACK_PACKAGE_VERSION_PATCH "0")
-set (CPACK_DEBIAN_PACKAGE_RELEASE 1)
-set (VERSION_SUFFIX "")
-set (PROJECT_VERSION "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}${VERSION_SUFFIX}")
-set (PROJECT_PACKAGE "xournalpp")
-set (PROJECT_STRING "${PROJECT_NAME} ${PROJECT_VERSION}")
-set (PROJECT_URL "https://github.com/xournalpp/xournalpp")
+project(xournalpp
+        VERSION 1.1.0
+        DESCRIPTION "Xournal++ - Open source hand note-taking program"
+        HOMEPAGE_URL "https://xournalpp.github.io/"
+        LANGUAGES CXX C)
+# according to https://cmake.org/cmake/help/latest/command/project.html fix suffix
```

Comment:
Do you want to introduce tweaks? This should be solved now as modifications to the version system will break the pipelines and all helper scripts.

Reply:
What do you mean? This does not introduce tweaks, since I don't set a tweak to something else than 0?

Reply:
For what is this comment then?

Reply:
Ah, to add our ~dev postfix, the hole logic after project is for that purpose.

Reply:
Then I don't get why that comment is there. Is it just a description for the following lines or is it actually a TODO? For me it reads like the latter.